CHANGELOG-hmp-181-remove-search-feature.md

```diff
@@ -0,0 +1,4 @@
+- Removed search feature from home page.
+- Added vertical padding to the count section.
+- Concatenated count to nearest thousands place. (10,5000 -> 10.5k)
```

Comment:
```suggestion
- Formatted 10k+ homepage entity counts to nearest thousands place. (10,500 -> 10.5k)
```

`Formatted` would be a clearer term to users/changelog readers IMO + there's a minor typo in the provided formatting example.